### PR TITLE
pkt/lwip: minor fixes

### PR DESCRIPTION
**Pull request by lebrush**
This fixes some guard comments and removes a duplicated line.

### CODE REVIEW

**Comment by OlegHahm** [comment]
Looks sensible. ACK

File: pkg/lwip/contrib/netdev2/lwip_netdev2.c [hunk 1]
```diff
@@ -149,7 +149,6 @@ err_t lwip_netdev2_init(struct netif *netif)
     netif->flags |= NETIF_FLAG_IGMP;
     netif->flags |= NETIF_FLAG_MLD6;
     netdev->context = netif;
-    netdev->event_callback = _event_cb;
```
Comment (miri64) [inline pkg/lwip/contrib/netdev2/lwip_netdev2.c]: Is this set at some other point? Why is this line removed?

Comment (lebrush) [inline pkg/lwip/contrib/netdev2/lwip_netdev2.c]: Yup [L:84](https://github.com/lebrush/RIOT/blob/9528b2220920c4d5abe5e737a6bf650e16281ed9/pkg/lwip/contrib/netdev2/lwip_netdev2.c#L84) :-)
